### ISSUE TITLE
Documentation regarding associate_eip for EC2 profiles

### ISSUE THREAD

**Issue by tbaker57** [saltstack/salt#27815]
The documentation 

https://github.com/saltstack/salt/blob/develop/doc/topics/cloud/aws.rst

regarding the associate_eip parameter seems to indicate that the value associated with this parameter should be an Elastic Network Interface (ENI) identifier:
```
      # Uncomment this to associate an existing Elastic IP Address with
      # this network interface:
      #
      # associate_eip: eni-XXXXXXXX
```
In fact, this should be an Allocation ID as per

https://github.com/saltstack/salt/blob/develop/salt/cloud/clouds/ec2.py#L1319

```

    retries = 5
    while retries > 0:
        params = {'Action': 'AssociateAddress',
                  'NetworkInterfaceId': eni_id,
                  'AllocationId': eip_id}

```
Perhaps the documentation should be updated to reflect the value should be of the form eipalloc-XXXX.

cheers,
Tim

**Comment by jfindlay** [comment]
@tbaker57, thanks for the report.  You are welcome to submit a pull request to fix this.

**Comment by rallytime** [comment]
Oh, I fixed this before I saw the comment above. This has been fixed on the 2015.8 branch with #27914.
